### ISSUE TITLE
你好，为什么我用自己的数据集训练之后，output/result_dir/eval_result.txt中都为零呢，eval_f = 0.0 eval_precision = 0.0 eval_recall = 0.0 global_step = 16856 loss = 0.0

### ISSUE THREAD

**Issue by mingxixixi** [ProHiryu/bert-chinese-ner#51]
你好，为什么我用自己的数据集训练之后，output/result_dir/eval_result.txt中都为零呢？
eval_f = 0.0 eval_precision = 0.0 eval_recall = 0.0 global_step = 16856 loss = 0.0
